dspace-api/src/main/java/org/orcid/utils/ReleaseNameUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.orcid.utils;
```

Comment:
This file needs a valid DSpace license header

[ERROR] Failed to execute goal com.mycila:license-maven-plugin:2.11:check (check-headers) on project dspace-api: Some files do not have the expected license header -> [Help 1]

Reply:
Another `org.orcid.utils.*` file that shouldn't be here.  I'll get it removed.